sklearn/linear_model/logistic.py

```diff
@@ -1606,7 +1660,24 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : object
         """
-        _check_solver_option(self.solver, self.multi_class, self.penalty,
+        if self.solver == 'default':
+            _solver = 'lbfgs'
```

Comment:
I find it strange to have these preceding undescores. This is just a plain old local variable.